ipatests/test_install/test_changeconf.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018  FreeIPA Contributors see COPYING for license
+
+from __future__ import absolute_import
+
+import pytest
+from ipaclient.install.ipachangeconf import IPAChangeConf
+
+
+@pytest.fixture(scope='session')
+def config_file(tmpdir_factory):
```

Comment:
``tmpdir_factory`` was introduced in pytest 2.8, but RHEL 7.5 has 2.7.0-2.el7. You need to use another way to create a temporary file, e.g. ``tmpdir`` fixture with a ``function`` scope fixture.

Reply:
Nice observation, would pr-ci nightly catch this eventually?

Reply:
I don't think so. QE will eventually run into this and complain.